src/test/java/core/basesyntax/service/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private final FileReaderService fileReaderService = new FileReaderServiceImpl();
+
+    @Test
+    public void read_validFilePath_ok() {
+        String validFilePath =
+                this.getClass().getClassLoader().getResource("FileReaderTestFile.csv").getPath();
+        List<String> expectedResultList = List.of(
+                "b,banana,0",
+                "b,apple,10",
+                "s,banana,15",
+                "p,banana,5",
+                "r,apple,20");
+        assertEquals(expectedResultList, fileReaderService.read(validFilePath));
```

Comment:
Also, create variable for actual result(fileReaderService.read(validFilePath)) and compare